packages/core/src/dialects/abstract/query.js

```diff
@@ -586,7 +584,6 @@ export class AbstractQuery {
         // The string prefix isn't actualy needed
         // We use it so keyPrefix for different keys will resolve to the same array if they have the same prefix
         // TODO: Find a better way?
-        $keyPrefixString = keyPrefixString(key, keyPrefixStringMemo);
```

Comment:
Looks like this was here on purpose, do you think this comment is still valid?

Reply:
`$keyPrefix` is being used but not `$keyPrefixString`. This was picked up by the eslint `no-unused-vars` rule.
The comment is still valid for `$keyPrefix`.